scripts/qsscheck.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+import argparse
+import fnmatch
+import os
+import os.path
+import re
+import sys
+import tinycss.css21
+import PyQt5.QtWidgets
+
+
+def get_skins(mixxx_path):
+    skins_path = os.path.join(mixxx_path, 'res', 'skins')
+    for entry in os.scandir(skins_path):
+        if entry.is_dir():
+            yield entry.name
+
+
+def make_glob(name):
+    return re.sub(r'<.*>', '*', name)
+
+
+def get_global_names(mixxx_path):
+    classnames = set()
+    objectnames = set()
+    for root, dirs, fnames in os.walk(os.path.join(mixxx_path, 'src')):
+        for fname in fnames:
+            ext = os.path.splitext(fname)[1]
+            if ext in ('.h', '.cpp'):
```

Comment:
(see below comment -- here's another place where we can save a level of indentation)

Reply:
How? We have an elseif case, too.